packages/flutter_arc_text/lib/src/arc_text_painter.dart

```diff
@@ -79,6 +95,13 @@ class ArcTextPainter {
       ..save()
       ..translate(offset.dx, offset.dy)
       ..rotate(_angleWithAlignment);
+
+    customDecoration?.call(
```

Comment:
Now I understand better ;)
But still, there's one case you cannot deal with here: I want to paint something on the text (like a background, but afterwards) (probably with alpha).

Reply:
That's true but for 80% of use cases (where you just want to paint generally over the area) you can just put in a stack.

We could of course have a second custom decoration painter which runs after the text has been painted but I feel you than can also switch to the [custom painter](https://github.com/ookami-kb/flutter_arc_text/discussions/14#discussioncomment-588459)  (which could probably shown as an example in the documentation). My goal is mainly to simplify the '80%' of decoration use cases: drawing something along the same circle but not overlapping with the text. Most other things (drawing inside the circle, drawing over the whole thing etc.) can easily be solved with stacks or parent widgets.

Reply:
I don't know if it's 80% of the decoration use cases, and I cannot figure out how to use stacks or parent widgets in order to paint a band on top of a hidden arc text.

This is my actual use case:
* I have a text that I want to display on an arc
* Just before I want to paint a blue band as the text background, starting 10° before, ending 10° after, with a padding of 15 around the text
* could you implement that as an example?

Of course the band hiding the text is definitely an advanced case.